ci/env/kava-protonet/genesis.json

```diff
@@ -2172,7 +2172,7 @@
         "max_deposit_period": "172800s"
       },
       "voting_params": {
-        "voting_period": "600s"
+        "voting_period": "604800s"
```

Comment:
is this being updated on purpose? it's more useful for our internal testnets to use short voting periods so that we can submit & pass test proposals (waiting 10min vs waiting 1 week)

Reply:
yeah, frontend requested this so its up longer for testing on webapp. I suggest we increase it for now and can revert it back later.